code/lib/source-loader/extract-source.d.ts

```diff
@@ -1,4 +1,4 @@
 // DEPRECATED
 // Do not import @storybook/source-loader/extract-source directly.
 // Import @storybook/source-loader instead.
-export * from './dist/types/extract-source.d';
\ No newline at end of file
+export * from './dist/index';
```

Comment:
This likely broke in the move to use tsup to bundle source-loader.  It's deprecated though, so not a big deal I think.

Reply:
Why index, instead of extract-source.d?